aspnet/DropDownList/Validation.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Validation in DropDownList control for Syncfusion ASP.NET WebForm
-description: Describes about the Validation in DropDownList control for Syncfusion ASP.NET WebForm
+title: Validation in  ASP.NET WebForm Dropdown List control | Syncfusion
+description: Learn here all about ports support in Syncfusion Essential ASP.NET Webforms Dropdown List Control, its elements, and more.
 platform: aspnet
 control: DropDownList
 documentation: ug
 keywords: Validation, DropDownList, dropdown, jQuery Validator, ASP Validator
 
 ---
 
-# Validation
+# Validation in  ASP.NET Web Form Dropdown List control
```

Comment:
Remove extra space and use Control not control

Reply:
resolved